test/examples/test_simple_gp_regression.py

```diff
@@ -185,6 +185,34 @@ def test_gp_posterior_mean_skip_variances_slow_cuda(self):
             self.assertTrue(torch.allclose(mean_skip_var, mean))
             self.assertTrue(torch.allclose(mean_skip_var, likelihood_mean))
 
+    def test_gp_posterior_single_training_point_smoke_test(self):
+        train_x, test_x, train_y, _ = self._get_data()
```

Comment:
why not use `self._get_data(num_data=1)`?

Reply:
Because I didn't read the `_get_data` method carefully enough :-)